src/coordinator/index.js

```diff
@@ -31,6 +35,74 @@ class Coordinator {
     this.roles = new Roles(sdk, request, baseUrl);
     this.users = new Users(sdk, request, baseUrl);
   }
+
+  /**
+   * Sets a selected oranization ID to be used in tenant based requests
+   *
+   * @param {string} organizationId the ID of the organization
+   *
+   * @example
+   * contxtSdk.coordinator
+   *   .setOrganizationId('36b8421a-cc4a-4204-b839-1397374fb16b');
+   */
+  setOrganizationId(organizationId) {
```

Comment:
When we do the next service.  Would we add those routes to coordinator?  Or would we update their service?

Perhaps we could move this up into the SDK where the clients for other services could use it.

Reply:
Talked in-person. Leaving as-is for now